components/backup-stream/src/router.rs

```diff
@@ -1191,13 +1196,16 @@ impl StreamTaskInfo {
                 .await?
                 .generate_metadata(store_id)
                 .await?;
+
+            fail::fail_point!("after_moving_to_flushing_files");
             crate::metrics::FLUSH_DURATION
                 .with_label_values(&["generate_metadata"])
                 .observe(sw.lap().as_secs_f64());
 
             // flush log file to storage.
-            self.flush_log(&mut metadata_info).await?;
-
+            self.flush_log(&mut metadata_info)
```

Comment:
It will be good to make error returned by `flush_log` brings what file it failed at.

Reply:
Then how about print metadata_info.file_groups here?